app/config/config.yml

```
@@ -15,6 +15,8 @@ parameters:
     env(PS_LOG_OUTPUT): "%kernel.logs_dir%/%kernel.environment%.log"
     mail_themes_uri: "/mails/themes"
     mail_themes_dir: "%kernel.project_dir%%mail_themes_uri%"
+    prestashop_temporary_image_directory: "%kernel.project_dir%/img/tmp/"
```

```suggestion
    image_temp_dir: "%kernel.project_dir%/img/tmp/"
```
be consistent with above parameters (like `mail_themes_dir`)